lib/xenopsd.ml

```diff
@@ -171,7 +171,7 @@ let main backend =
       ~rpc_fn
       () in
 
-  (** we need to catch this to make sure at_exit handlers are
+  (* we need to catch this to make sure at_exit handlers are
```

Comment:
This could also be linked to the `signal_handler` by deleting the empty line perhaps.